glide.lock

```diff
@@ -223,7 +223,7 @@ imports:
   - transport
 - name: gopkg.in/src-d/go-errors.v0
   version: 0c303ec4c027302259ec1738f19f515aad25f13f
-- name: gopkg.in/src-d/simple-linguist.v1
+- name: gopkg.in/src-d/enry.v1
```

Comment:
@alcortesm We cannot do this because all the internal enry imports are using gopkg.in/src-d/enry.v1 .